loadbalancer/errors.go

```diff
@@ -0,0 +1,97 @@
+package loadbalancer
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/libp2p/go-libp2p-core/protocol"
+)
+
+// ErrAlreadyRegistered indicates a protocol has already been registered as a route
+type ErrAlreadyRegistered struct {
```

Comment:
It looks like several of the errors in this file are no longer used. Can we remove them?